main.py

```diff
@@ -266,6 +266,10 @@ def get_best_checkpoint_path(args, phase, task_name=None):
             checkpoint = glob.glob(
                 os.path.join(args.run_dir, task_name, "model_state_target_train_val_*.best.th")
             )
+            if len(checkpoint) == 0:
```

Comment:
Why is this in this PR?

Reply:
Make sure this matches the comment header.

Reply:
Yeah I'll put this in a separate PR to do with diagnostics.